.github/workflows/latest_import.yml

```diff
@@ -0,0 +1,129 @@
+name: Weekly late importers
+
+on:
+  schedule:
+    - cron: '0 4 * * 1'  # Run at 04:00 UTC every Monday
+  workflow_dispatch:
+
+jobs:
+  late-importers:
+    name: Build
+    runs-on: pr
+    steps:
+    - name: cleanup
+      run: |
+        find . -name . -o -prune -exec rm -rf -- {} +
+        # Delete all but the 5 most recent toolchains.
+        # Make sure to delete both the `~/.elan/toolchains/X` directory and the `~/.elan/update-hashes/X` file.
+        # Skip symbolic links (`-type d`), the current directory (`! -name .`), and `nightly` and `stable`.
+        if cd ~/.elan/toolchains && find . -maxdepth 1 -type d ! -name . -print0 | xargs -0 ls -1td | grep -v 'nightly$' | grep -v 'stable$' | tail -n +6 | xargs -I {} sh -c 'echo {} && rm -rf "{}" && rm "../update-hashes/{}"'; then
+            : # Do nothing on success
+        else
+            : # Do nothing on failure, but suppress errors
+        fi
+
+    # The Hoskinson runners may not have jq installed, so do that now.
+    - name: 'Setup jq'
+      uses: dcarbone/install-jq-action@v2.1.0
+
+    - name: install elan
+      run: |
+        set -o pipefail
+        curl -sSfL https://github.com/leanprover/elan/releases/download/v3.1.1/elan-x86_64-unknown-linux-gnu.tar.gz | tar xz
+        ./elan-init -y --default-toolchain none
+        echo "$HOME/.elan/bin" >> "${GITHUB_PATH}"
+
+    - uses: actions/checkout@v4
+      with:
+        ref: 'master'
+
+    - name: If using a lean-pr-release toolchain, uninstall
+      run: |
+        if [[ $(cat lean-toolchain) =~ ^leanprover/lean4-pr-releases:pr-release-[0-9]+$ ]]; then
+          printf 'Uninstalling transient toolchain %s\n' "$(cat lean-toolchain)"
+          elan toolchain uninstall "$(cat lean-toolchain)"
+        fi
+
+    - name: print lean and lake versions
+      run: |
+        lean --version
+        lake --version
+
+    - name: add minImport linter option
```

Comment:
Just to check my understanding: because of this option, getting the cache with lake would be useless, right?

Reply:
Yes, if the cache was available, `lake build` would replay it, since setting options on the command line does not seem to trigger a rebuild.  Moreover, once you modify the `lakefile`, everything has to be rebuild.